docs/articles/viewing-epoch-based-metrics.rst

```diff
@@ -0,0 +1,232 @@
+.. _coreapi-epoch-metrics-howto:
+
+##############################################
+ How to View Epoch-Based Metrics in the WebUI
+##############################################
+
+.. meta::
+   :description: Learn how to analyze and visualize training progress and validation performance over multiple epochs using the Core API.
+   :keywords: CoreAPI, WebUI, epochs, metrics, metric data
+
+Sometimes, you want to analyze and visualize your model's training progress and validation
+performance over multiple epochs.
+
+In this article, we'll show you how to view epoch-based metric data in the WebUI by reporting an
+epoch metric to the Determined master via the Core API. To do this, we'll define an epoch metric.
+This metric is then used as the X-Axis label in the WebUI.
+
+**Recommended**
+
+-  :doc:`Quickstart for Model Developers <../tutorials/quickstart-mdldev>`
+-  :doc:`Core API User Guide <../model-dev-guide/apis-howto/api-core-ug>`
+
+**********************************
+ Set Up Your Training Environment
+**********************************
+
+To begin, you'll need a Determined cluster. If you are new to Determined AI (Determined), you can
+install the Determined library and start a cluster locally.
+
+-  Ensure you have Docker running and then run the following command:
+
+.. code::
+
+   pip install determined
+
+   # If your machine has GPUs:
+   det deploy local cluster-up
+
+   # If your machine does not have GPUs:
+   det deploy local cluster-up --no-gpu
+
+.. include:: ../_shared/note-pip-install-determined.txt
+
+****************************
+ Step 1: Run the Experiment
+****************************
+
+Before reporting metrics, we'll run our experiment and ensure we can see it in the WebUI.
+
+.. note::
+
+   To follow along with the steps shown in this article, you'll need to download the Core API
+   PyTorch MNIST Tutorial files :download:`core_api_pytorch_mnist.tgz
+   </examples/core_api_pytorch_mnist.tgz>`. You can also find the files by visiting the `Github
+   repository
+   <https://github.com/determined-ai/determined/tree/master/examples/tutorials/core_api_pytorch_mnist>`_.
+
+For this step, we'll use our ``model_def.py`` script and its accompanying ``const.yaml`` experiment
+configuration file.
+
+From the directory containing our files, we'll begin by running the following command:
+
+.. code:: bash
+
+   det e create const.yaml . -f
+
+We don't have any data to plot yet, but we'll visit the Determined WebUI to see that our experiment
+is running.
+
+To do this, we'll navigate to ``http://localhost:8080/``.
+
+To sign in, we'll accept the default determined username, leave the password empty, and click **Sign
+In**.
+
+In the WebUI, we'll select our experiment, and then navigate to the **Logs** tab.
+
+************************
+ Step 2: Report Metrics
+************************
+
+In this section, we'll define our epoch metric. We'll also report training and validation metrics to
```

Comment:
Maybe this could be shortened to just "In this section, we'll define our epoch metric."

And then "To follow along, use..."

Reply:
resolved